CSS/style.css

```diff
@@ -193,3 +193,164 @@
 
 }
 
+
+.third-section {
+    min-width: 1440px;
+    height: 896px;
+    background-color: #D5FAFC;
```

Comment:
- [ ]  What you can do is remove `background-color` property here and add it to its parent `div` for example you can name its parent div `.third-section-container`
- [ ] Instead of `min-width` make it `max-width` .
- [ ] Also You can add `margin: auto` here to center `.third-section` inside the `.third-section-container`

In the picture below you can see the red border is `.third-section-container` and green border is `.third-section` so its centered inside of it
![sections](https://user-images.githubusercontent.com/58483864/220070452-bf9cc2e4-48be-462a-80f2-5a4f3cce398c.png)